src/topology/metric_space/thickened_indicator.lean

```diff
@@ -5,6 +5,7 @@ Authors: Kalle Kytölä
 -/
 import data.real.ennreal
 import topology.continuous_function.bounded
+import measure_theory.integral.bochner
```

Comment:
I don't think you should import advanced measure theory in a purely topological file like this one. Instead, you can create a new file in the measure theory folder (or put the two relevant lemmas in a file where it makes sense, like `borel_space.lean`)

Reply:
`borel_space.lean` does not import `lintegral`. I will try placeing these in `set_integral.lean`, unless there is a more appropriate file.